app/config/featureswitch/FeatureSwitching.scala

```diff
@@ -16,17 +16,13 @@
 
 package config.featureswitch
 
-import config.FrontendAppConfig
-
 trait FeatureSwitching {
 
-  val appConfig: FrontendAppConfig
-
   val FEATURE_SWITCH_ON = "true"
   val FEATURE_SWITCH_OFF = "false"
 
   def isEnabled(featureSwitch: FeatureSwitch): Boolean =
-    (sys.props.get(featureSwitch.name) orElse appConfig.conf.getString(featureSwitch.name)) contains FEATURE_SWITCH_ON
+    sys.props get featureSwitch.name contains FEATURE_SWITCH_ON
```

Comment:
We need to keep how this used to work
get feature switch state from system properties, if it's not there, get from config